lib/translator/inquiry.cc

```diff
@@ -187,6 +231,59 @@ void TranslateUnitSerialNumberVpd(
   memcpy(&buffer[sizeof(result)], &product_serial_number, result.page_length);
 }
 
+// Returns Extended Inquiry Data Page. Refer to section 6.1.5
+// https://nvmexpress.org/wp-content/uploads/NVM_Express_-_SCSI_Translation_Reference-1_5_20150624_Gold.pdf
+void TranslateExtendedInquiryDataVpd(
+    const nvme_defs::IdentifyNamespace &identify_namespace_data,
+    const nvme_defs::IdentifyControllerData &identify_controller_data,
+    absl::Span<uint8_t> buffer) {
+  uint8_t dps = 0;
+  dps |= identify_namespace_data.dps.reserved4;
+  dps |= identify_namespace_data.dps.md_start << 1;
+  dps |= identify_namespace_data.dps.pit << 2;
+
+  uint8_t dps_result = (dps == 0b000) ? 0b0 : 0b1;
```

Comment:
`dps_result = dps` is equivalent

Reply:
can change this whole block of code to
`dps_result = identify_namespace_data.dps.md_start || identify_namespace_data.dps.pit`